README.md

```diff
@@ -1,70 +1,266 @@
+
+## Usage
+
+### Getting Started
+
+#### Required Options
+Info you need to find/decide on to successfully build our Stack
+
+-   Required Params to Fill Out
+    -   HostedZone; A Domain Setup in Route53 [Guide](http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingHostedZone.html)
+    -   SSLCertificateARN; See the [SSL Setup](#ssl-setup) Section
+    -   LicenseCount; How many license you have purchased from Chef. (25 is the default, this is the amount of default "free" nodes you get starting off)
+    -   DBUser; Username for DB (Required for Existing or New Setups)
+    -   DBPassword; Password for DB (Required for Existing or New Setups)
+-   Required Params to Select
+    -   ChefSubdomain; See [Blue/Green](#bluegreen-deployment) Section for more Info
+    -   ChefServerVersion; Select Version to install, versions listed are the only ones that have packages for Ubuntu Xenial
+    -   ManageVersion; Select Version to install, versions listed are the only ones that have packages for Ubuntu Xenial
+    -   ReportingVersion; Select Version to install, versions list are the only ones that support ElasticSearch
+    -   DBMultiAZ; Select if you want the DB setup in Multiple Availability Zones
+    -   ElasticInstanceType; Default is t2.small.elasticsearch, good for small or testing situations
+    -   KeyName; Select SSH Key
+    -   VPC; Select VPC to Build in
+    -   SSHSecurityGroup; Select group for SSH access
+
+#### Options for Larger Setups
+**Estimated** These are the items to best look when looking at larger node check-ins (5k+ ish)
+
+-   Params to Change only if 4-5k+ Nodes (Estimated; still need to do actual load tests)
+    -   DBInstanceType; Default (t2.large) is the minimum required for Chef DB Connections
+    -   ESProxyInstanceType; Default is good for small-medium Chef Build Outs. Change as needed
+    -   ElasticInstanceType; Default is probably not going to cut it for tons of requests
+    -   DBMultiAZ; Would recommend setting this to true
+
+#### Optional Setup
+
+-   Restoring Setups; See the [Restore/Backup](#restorebackup-options) Section
+    -   **Important:**  If intending to do external knife ec backups, check this section before setting up.
+-   NewRelic Setup; See the [NewRelic](#new-relic) Section
+-   Sumologic Setup; See the [Sumologic](#sumologic) Section
+
+##### Mail Server Setup
+If you fill out the following Params the script will setup a postfix relay to allow the Chef Server to send email.
+
+-   MailCreds; if the format `$username:$password`
+-   MailHost; Such as mailgun.org
+-   MailPort; Mail server port
+
+##### Development Selections
+
+Options that don't need to be bothered with except in development situations.
+
+See [Contributing](#contributing) for how to contribute to this project
+
+-   GitBranch; sets branch to pull scripts from
+    -   Currently only affects NewRelic and Sumologic
+
+### SSL Setup
+
+To simplify the setup only the ELB (Public Side) is setup with SSL.
```

Comment:
No, it's just information. Not recommending for anyone what they should choose. Just what info they need and some basic help if they have no idea what they need.